wai-aria/scripts/aria-utils.js

```diff
@@ -157,5 +163,35 @@ const AriaUtils = {
   },
 
 
+  /*
+  Tests computed LABEL and ROLE of all elements matching selector using existing
+    verifyLabelsBySelector(), verifyRolesBySelector() functions and passes a test name prefix
+    to ensure uniqueness.
+
+  Ex: <div aria-label="foo" role="button"
+        data-testname="div with role=button is labelled via aria-label"
+        data-expectedlabel="foo"
+        data-expectedrole="button"
+        class="ex">
+
+      AriaUtils.verifyRoleAndLabelBySelector(".ex")
+
+  */
+  verifyRoleAndLabelBySelector: function(selector) {
+    let labelTestNamePrefix = "(Label Test) ";
+    let roleTestNamePrefix = "(Role Test) ";
```

Comment:
Verbosity nit: everything in WPT is a test.
```suggestion
    let labelTestNamePrefix = "Label: ";
    let roleTestNamePrefix = "Role: ";
```